### DISCUSSION TITLE
Fruits - Kuebiko and Immortal Pear

### DISCUSSION THREAD

**Discussion by brian-man** [sunflower-land/sunflower-land#2010] [General]
**Fruit regrowth is now the same rate as fruit tree planting**

Originally the plant time was greater than the replenish time.  This meant that it was faster to harvest a tree for the 2nd+ fruit[s] vs. the 1st fruit[s].

Consequently, the Immortal Pear is compelling for two reasons [a] fewer seeds are required over time and [b] you get more "not first" fruit[s].
```
  "Immortal Pear": {
    description: "A long-lived pear that makes fruit trees last longer.",
    ingredients: {
      Gold: new Decimal(25),
    },
    boost: "+1 harvest",
  },
```

With this change...
https://github.com/sunflower-land/sunflower-land/pull/2000
> This PR standardises the times so the plant and replenish times are the same.

because

> The differing replenish and initial growth times were confusing for users.

Reason [b] for Immortal Pear is effectively removed.

For owners of Kuebiko, reason [a] never applied (other than for supply syncing reasons`*`).  In fact, such owners are better off *without* the Immortal Pear because they get to harvest wood more often.

`*` Note that the sync-for-axes reason vanishes for owners of Foreman Beaver.

Is the "Immortal Pear is a debuff if you have Kuebiko" effect intentional?

**Comment by adamhannigan** [comment]
Hi @brian-man can you please elaborate on point 'b' - `you get more "not first" fruit[s].` - I don't understand what you mean by "not first fruits".

It is an interesting point you make. If a user has a both a Kuebiko and a Foreman Beaver, then having a Immortal Pear would make it harder for them to get free wood (only for Apple Trees and Orange trees - You still need axes for Blueberry bushes)

Having a Kuebiko by itself means that an Immortal Pear has a neutral effect. It isn't a debuff, but there is no point in crafting it.

Utility wise, for people with a Kuebiko it would be pointless to have the Immortal pear (unless they want the decoration).

The intention of the Immortal Pear is to provide a bonus round of harvests for fruit pickers.

**Reply by brian-man** [reply]
> I don't understand what you mean by "not first fruits".
Sure, my phrasing was weird because English isn't as good as math sometimes. :)

The "first fruit" is the first harvest from a planted tree.  The "not first fruit" is the 2nd, 3rd, etc. time you use your basket or whatever to collect and reset the timer for the next harvest or end-of-tree.

> Having a Kuebiko by itself means that an Immortal Pear has a neutral effect. It isn't a debuff, but there is no point in crafting it.

It's a debuff in the sense that you get the axe->wood conversion less often.  So effectively you get one less tree (periodically) than you would without it.
